tests/data/conda_format_repo/osx-64/repodata.json

```diff
@@ -25,10 +25,10 @@
       "depends": [],
       "license": "zlib",
       "license_family": "Other",
-      "md5": "5eab2de4933e1914f86d1586a8a40c39",
+      "md5": "a1b03ba4ff40c83e4eb9c8fae40315a7",
       "name": "zlib",
-      "sha256": "9b73eb4e3197eccb485250709c6f4782a83a8688d5b8251a212251c451d9b9ad",
-      "size": 106948,
+      "sha256": "857293f389b188ba1fb02401a873fca5098db612ce6646c684efee77ad807f77",
```

Comment:
I notice that previously the sha256 sum incorrectly matched that of the `.tar.bz2` version.